src/plays/simple-calculator/SimpleCalculator.js

```diff
@@ -0,0 +1,22 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import React from 'react';
+import Calculator from './components/Calculator';
+
+// WARNING: Do not change the entry componenet name
+function SimpleCalculator(props) {
+  // Your Code Start below.
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```

Comment:
Add an additional and unique class name here, say `soni-calc`. We will use it in the CSS files.

Reply:
Hi @suryanshsoni120, please add the unique class name here and use it to prefix in your CSS file.